java/vector/pom.xml

```diff
@@ -50,6 +50,12 @@
       <artifactId>commons-codec</artifactId>
       <version>1.10</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.arrow</groupId>
+      <artifactId>arrow-memory-netty</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Maybe we also need the memory-unsafe dependency here? 
Otherwise, our integeration tests will not be able to pass.

Reply:
I don't think the integration tests use UnsafeAllocator do they?

Reply:
I ran `mvn clean install -Pintegration-tests` and it was clean. So the integration tests went ok w/ Netty

Reply:
Thanks for your feedback. 
It is weird. I thought only unsafe allocator supports allocating a buffer larger than 2GB, because the netty allocator uses int32 internally.

Reply:
I believe the netty allocator can allocate 64-bit spaces but netty buffers can't directly reference them?

Reply:
Maybe we should run the unit tests once for each of the 2 allocators?